linchpin/InventoryFilters/InventoryProviders.py

```diff
@@ -20,6 +21,7 @@
     "beaker": BeakerInventory,
     "duffy": DuffyInventory,
     "dummy": DummyInventory,
+    "nummy": NummyInventory,
```

Comment:
@14rcole Interesting approach I think i might have to consider your suggestion and re implement it. However , There might be the repurcussion of nummy being considered as the dummy provider and group it as a dummy provider again . :+1:  I will try to re implement it

Reply:
@14rcole : I have checked out the same there will be multiple places we have to update the nummy references . In order to avoid deduplication we might have to implement another pattern or make dummy more generic in nature . Which needs more work . 
I would suggest it as a another card not as part of this one .